@alias/commitlint/cli.test.js

```diff
@@ -1,3 +1,4 @@
+import {test, expect} from 'vitest';
```

Comment:
I don't know whether it is the preferred way for @escapedcat, but I'm totally fine with this change personally.

Reply:
Switchting to vitest? Sure, go for it! ❤️

Reply:
Nope, I mean global `test`, `expec` or import them manually.

I was using https://github.com/unplugin/unplugin-auto-import previously.

Reply:
Ah, I don't mind using auto-import

Reply:
Let's keep this PR as-is, and figure  out the best way to use vitest later.